### PR TITLE
[BASTION]: Jumphost module

### PR DESCRIPTION
**Pull request by artem-lifshits**
Simple bastion module

### CODE REVIEW

File: examples/bastion/default_cloud_init.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+mkdir -p /etc/sslcerts/live
+#generate Diffie-Hellman for TLS
+sudo openssl dhparam -out /etc/sslcerts/live/dhparams.pem 2048
+
+#enable port forwarding
+echo 'net.ipv4.ip_forward=1' | sudo tee -a /etc/sysctl.conf
+sudo sysctl -p /etc/sysctl.conf
+
+#configure SNAT
+sudo iptables -t nat -A POSTROUTING -o eth0 -s "${cidr}" -j SNAT --to "${bastion_address}"
```
Comment (anton-sidelnikov) [inline examples/bastion/default_cloud_init.sh:25]: end line

File: examples/bastion/main.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+resource "opentelekomcloud_vpc_subnet_v1" "subnet_1" {
+  name              = "simple-subnet"
+  description       = "some description"
+  cidr              = "192.168.55.0/24"
+  gateway_ip        = "192.168.55.1"
+  vpc_id            = opentelekomcloud_vpc_v1.vpc_1.id
+  availability_zone = "eu-de-02"
+  ntp_addresses     = "10.100.0.33,10.100.0.34"
+}
```
Comment (anton-sidelnikov) [inline examples/bastion/main.tf:29]: please add output here, because module outputs not printed

File: modules/bastion/locals.tf [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/*=================================
+LOCALS
+==================================*/
+locals {
+  sg-list = tolist(var.sg-rules)
```
Comment (anton-sidelnikov) [inline modules/bastion/locals.tf:5]: why dash here?

File: examples/bastion/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+| Name                                                                                           | Version   |
+| ---------------------------------------------------------------------------------------------- |-----------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform)                      | >= 0.13.0 |
+| <a name="requirement_opentelekomcloud"></a> [opentelekomcloud](#requirement\_opentelekomcloud) | >= 1.23.9 |
+
+## Modules
+
+| Name                                                  | Source                                                      | Version |
+|-------------------------------------------------------|-------------------------------------------------------------|---------|
+| <a name="module_dns"></a> [bastion](#module\_bastion) | "opentelekomcloud/modules/opentelekomcloud/modules/bastion" | 0.0.1   |
```
Comment (anton-sidelnikov) [inline examples/bastion/README.md:34]: version 0.0.3 next from current

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: modules/bastion/README.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+You can find different documentations versioned by terraform registry [here](https://registry.terraform.io/modules/opentelekomcloud/modules/opentelekomcloud/latest).
+
+## Terraform versions
+
+Terraform 0.13 or higher.
+
+## Usage
+
+```hcl
+resource "opentelekomcloud_vpc_v1" "vpc_1" {
```
Comment (vladimirvshivkov) [inline modules/bastion/README.md:19]: would be nice to use here new vpc module

File: examples/bastion/default_cloud_init.sh (hunk 1 above)
Comment (artem-lifshits) [reply examples/bastion/default_cloud_init.sh]: fixed

File: examples/bastion/main.tf (hunk 2 above)
Comment (artem-lifshits) [reply examples/bastion/main.tf:29]: Output added

File: modules/bastion/locals.tf (hunk 3 above)
Comment (artem-lifshits) [reply modules/bastion/locals.tf]: fixed

**Review by anton-sidelnikov — DISMISSED** [review]

File: modules/bastion/README.md (hunk 5 above)
Comment (artem-lifshits) [reply modules/bastion/README.md:19]: vpc module is now used in examples instead of resource

**Review by anton-sidelnikov — APPROVED** [review]

**Review by vladimirvshivkov — APPROVED** [review]

**Pull request merged**